tasks/main.yml

```diff
@@ -16,6 +16,36 @@
     name: "{{ package_names }}"
     state: present
 
+- name: Configure AppArmor for Clamd
+  when: "ansible_apparmor.status == 'enabled' and apparmor_complain"
+  block:
+    - name: Configure clamd in apparmor
+      ansible.builtin.lineinfile:
+        firstmatch: true
+        line: "/usr/sbin/clamd flags=(complain) {"
+        path: "{{ apparmor_clamd_configuration_path }}"
+        search_string: "/usr/sbin/clamd {"
+        state: present
+
+    - name: Configure freshclam in apparmor
+      ansible.builtin.lineinfile:
+        firstmatch: true
+        line: "/usr/bin/freshclam flags=(attach_disconnected,complain) {"
+        path: "{{ apparmor_freshclam_configuration_path }}"
+        search_string: "/usr/bin/freshclam flags=(attach_disconnected) {"
```

Comment:
Can you test whether it's possible to remove the quotes around the `line` and `search_string` values?  I _think_ they can be removed, but with the trailing left brace I'm not sure.

Reply:
@jsf9k  Removed the quotes and tested. Everything worked as expected. Thanks for the feedback.